ios/brave-ios/Tests/ClientTests/Resources/scripts/cosmetic-filter-tests.js

```diff
@@ -95,6 +97,14 @@
         results.delayedHasTextHidden = nodeDisplay === 'none'
       }
 
+      if (node.id === 'procedural-filter-child-node-id') {
+        const nodeDisplay = window.getComputedStyle(node).display
+        // 2 elements have this id, we want to test both their display values
```

Comment:
maybe a nit, but `id` is supposed to be unique in a page - either using `class` or renaming one might be better here

Reply:
Fair point, I think best to fix that in the PR now. Renaming would break the single rule matching multiple elements case so I've the elements to use a class instead in these tests and updated the rules to match. Similar update to the testplan & test site to align. [42b6297](https://github.com/brave/brave-core/pull/29139/commits/42b629733f46aadb196620348ebf7896fdb663eb)